modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -89,78 +89,298 @@ This allows your `kubeconfig` file to be consumed from inside the running contai
 
 [id="cnf-performing-end-to-end-tests-running-the-latency_tests"]
```

Comment:
module ids should have _{context} at the end of them, so that they can pick up a context for reuse. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata